src/gsi/cplr_get_fv3_regional_ensperts.f90

```diff
@@ -460,6 +574,155 @@ subroutine general_read_fv3_regional(this,fv3_filenameginput,g_ps,g_u,g_v,g_tv,g
 
   return       
   end subroutine general_read_fv3_regional
+
+  subroutine general_read_fv3_regional_dirZDA(this,fv3_filenameginput,g_ql,g_qi,g_qr,g_qs,g_qg,g_qnr,g_w)
+  !$$$  subprogram documentation block
+  !     firstly copied from general_read_arw_regional           .      .    .                                       .
+  ! subprogram:    general_read_fv3_regional_dirZda  read fv3sar model ensemble members
+  !   prgmmr: Ting             org: emc/ncep            date: 2018
+  !
+  ! abstract: read ensemble members from the fv3 model netcdf format (dynamic and tracer)
+  !           for use with hybrid ensemble option.
+  !
+  ! program history log:
+  !   2018-  Ting      - intial versions  
+  !   2019-03-13  CAPS(C. Tong)   - Port direct radar DA capabilities
+  !   2021-05-05  CAPS(J. Park)   - Modified to read hydrometeors and w only
+  !                                 
+  !   input argument list:
+  !
+  !   output argument list:
+  !
+  ! attributes:
+  !   language: f90
+  !   machine:  ibm RS/6000 SP
+  !
+  !$$$ end documentation block
+  
+      use netcdf, only: nf90_nowrite
+      use netcdf, only: nf90_open,nf90_close
+      use netcdf, only: nf90_inq_dimid,nf90_inquire_dimension
+      use netcdf, only: nf90_inq_varid,nf90_inquire_variable,nf90_get_var
+      use kinds, only: r_kind,r_single,i_kind
+      use gridmod, only: eta1_ll,eta2_ll
```

Comment:
Both eta1_ll and eta2_ll have been identified as unused.  Please remove unused variables.

Reply:
We deleted them as you suggested.

https://github.com/NOAA-EMC/GSI/pull/142/files#diff-86aea35b64364db7913dd1eb935d98f572e0601c925defcb5595748644029235R611